frontend/src/app/containers/HomePageWithAddon.js

```diff
@@ -44,9 +51,20 @@ export default class HomePageWithAddon extends React.Component {
 
         <div className="pinstripe responsive-content-wrapper"></div>
         <div className="responsive-content-wrapper">
-          <div data-hook="experiment-list">
-            <ExperimentCardList {...this.props} eventCategory="HomePage Interactions" />
-          </div>
+          <ExperimentCardList {...this.props} experiments={currentExperiments} eventCategory="HomePage Interactions" />
+          {pastExperiments.length > 0 && !showPastExperiments &&
+          <div className={classnames(['button', 'outline'])}
+              style={{ margin: '0 auto', display: 'table' }}
+              onClick={() => this.setState({ showPastExperiments: true })}
+              data-l10n-id="viewPastExperiments">View Past Experiments</div>}
+          {showPastExperiments &&
+          <div>
+            <div className={classnames(['button', 'outline'])}
+                style={{ margin: '0 auto', display: 'table' }}
+                onClick={() => this.setState({ showPastExperiments: false })}
+                data-l10n-id="hidePastExperiments">Hide Past Experiments</div>
+            <ExperimentCardList {...this.props} experiments={pastExperiments} />
```

Comment:
Getting a missing required prop warning here for the metrics `eventCategory`: 
```
Warning: Failed prop type: Required prop `eventCategory` was not specified in `ExperimentRowCard`.
```
Do we want to reuse "HomePage Interactions" here, or do we want to measure this differently?